lib/isolator.go

```diff
@@ -64,12 +65,24 @@ func (a *ACBuild) AddIsolator(name string, value []byte) (err error) {
 		if s.App == nil {
 			s.App = newManifestApp()
 		}
+		_, ok := types.LinuxIsolatorNames[*acid]
```

Comment:
I'm not certain if the linux isolators make it into there. I just switched it over and now the test that I added with this PR fails with:

```
=== RUN   TestAddIsolator
--- FAIL: TestAddIsolator (0.02s)
	isolator_test.go:55: non-zero exit code of 1: stdout:
		
		stderr:
		isolator add: unknown isolator name: os/linux/capabilities-retain-set
		
	
```